apps/adresse-service/src/main/java/no/nav/adresse/service/exception/BadRequestException.java

```diff
@@ -0,0 +1,11 @@
+package no.nav.adresse.service.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.client.HttpClientErrorException;
+
+public class BadRequestException extends HttpClientErrorException {
```

Comment:
Dette er vel en exception som finnes fra før av?

Reply:
Greia med custom exception er at det gir eget point of entry i @ControllerAdvice. Det løser problemet med egen @StatusCode